arkane/ess/qchem.py

```diff
@@ -264,8 +264,8 @@ def load_conformer(self, symmetry=None, spin_multiplicity=0, optical_isomers=Non
                         rot.append(rotation)
```

Comment:
This isn't your code but why is `rot.append(rotation)` not in the for-loop? Only the rotation for `i=2` gets used.

Reply:
I'll make a new PR for this